src/css/style.css

```diff
@@ -739,8 +739,8 @@ h2.final-ticker {
 }
 
 .image-constraint {
-  height: 60vh;
-  width: 40vw;
+  max-height: 60vh;
+  width: 30vw;
```

Comment:
I'm guessing you calibrated this to accommodate the images we have. Was trying to work out whether the max height would ever be an issue, like if there was a tall thin image?

Reply:
good question potentially someone could just delete the height restriction if it becomes an issue, We need to fix the width or the image starts to resize when it hits the edge but that does risk getting those invisible bits of `div` that extend past the image and make it very confusing to interact with on a very skinny image.

To do this perfectly we'd need some meta data about the image passed into the draggable area, or to start cropping the image but that's not ideal either.